### PR TITLE
fix typo

### LINKED ISSUE #100: Fix typo in Environment page of station

### PR DESCRIPTION
**Pull request by Rashmil-1999**
Fixes typo in the word `Celsius`

### CODE REVIEW

File: src/utils/format.ts [hunk 1]
```diff
@@ -22,12 +22,12 @@ export const decimalFormat = (value: number, decimals = 2): string => {
     return formatter.format(value);
 };
 
-export const convertFahrenheitToCelcius = (temp: number): string => {
-    const formattedCelciusTemp = decimalFormat((temp - 32) * (5 / 9));
-    return formattedCelciusTemp;
+export const convertFahrenheitTosius = (temp: number): string => {
```
Comment (navarroc) [inline src/utils/format.ts]: Typo here, should be I think convertFahrenheitToCelsius

Comment (Rashmil-1999) [reply src/utils/format.ts]: This is weird, this didn't comeup when I searched for celcius ... it seems the search was case sensitive... let me correct this...

Comment (Rashmil-1999) [reply src/utils/format.ts]: ahh I think I omitted out 'Cel' ... :P

**Comment by navarroc** [comment]
Is there a way to deploy this on dev so we can verify the fix?

**Comment by navarroc** [comment]
This change looks good, but it would be good to test it out on dev before we merge and make a new release.

**Review by navarroc — APPROVED** [review]
Looks good

**Pull request merged**